web/src/i18n/translations/en.yml

```
@@ -1,582 +1,585 @@
-Successfully created webhook!: ''
-Failed to delete webhook.: ''
-Successfully deleted webhook!: ''
-Failed to update webhook.: ''
-Successfully updated webhook!: ''
-Failed to create integration.: ''
-Successfully created integration!: ''
-Failed to create project.: ''
-Successfully created project!: ''
-Failed to delete workspace.: ''
-Successfully deleted workspace!: ''
+Copy: ''
```

YAMLlint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)